rotkehlchen/chain/substrate/typing.py

```diff
@@ -10,6 +10,23 @@
 BlockNumber = NewType('BlockNumber', int)
 
 
+class SubstrateOwnNodeName(Enum):
```

Comment:
What is this type for? Why on earth would you ever make an Enum with 1 option? I don't get this ... what are you doing here?

This is just another node to connect to, so it should be part of the `KusamaNodeName` Enum.

Reply:
My point of view was that "OWN" should be a singleton or a unique object, and not another option per each substrate chain we support. I've implemented it as an Enum for convenience: same methods and it behaves like the other public node options.

A specific OWN option per chain Enum (e.g. KusamaNodeName, PolkadotNodeName) requires to extend any node logic for taking into account the chain the instance is connected to (if Kusama check/use KusamaNodeName.OWN, if Polkadot check/use PolkadotNodeName.OWN). In the current implementation the instance does not need to know which chain is connected to.

I will change it though

Reply:
It wouldn't be KusamaNodeName, PolkadotNodeName. Just `NodeName` or `SubstrateNodeName`. This extra class is just not required.